charts/emissary-ingress/values.yaml.in

```diff
@@ -395,9 +395,9 @@ agent:
   rpcAddress: https://app.getambassador.io/
   createArgoRBAC: true
   image:
-    # Leave blank to use image.repository and image.tag
-    tag:
-    repository:
+    tag: 0.0.7
```

Comment:
Tele asks saas_app for <latest_version> that matches <this_tele_version> https://github.com/datawire/saas_app/blob/master/internal/app/system-a/rpc/server.go#L196-L263

Reply:
the latest version is stored in a text file in a s3 bucket. this text file is update by the agent release process.

For clarity, these comments are about the telepresence agent. That's not going to get confusing 🤷

Reply:
Using pure helm, we could simply rely on helm's built in [chart dependancies](https://helm.sh/docs/chart_best_practices/dependencies/). This could, for example, automatically update for new patch versions but require manual updating for new minor versions

Reply:
Chart dependencies is IMO the ideal solution. But since the agent doesn't have its won chart yet, I don't think that needs to happen in _this_ PR.

Reply:
Yeah, if we had more of an operator pattern for managing it then something like that would be possible but a lot of times these manifest and helm values charts are checked into customers gitops process so it wouldn't have the ability to call out to grab that information.